authors/apps/authentication/views.py

```diff
@@ -1,24 +1,33 @@
 import os
 import jwt
 from datetime import datetime, timedelta
+
 from django.conf import settings
 from django.template.loader import render_to_string
 from django.core.mail import send_mail
+from django.db import IntegrityError
 from rest_framework import status, generics
 from rest_framework.permissions import AllowAny, IsAuthenticated
 from rest_framework.response import Response
 from rest_framework.views import APIView
-from rest_framework.renderers import BrowsableAPIRenderer
-from .renderers import UserJSONRenderer
+from social_django.utils import load_strategy, load_backend
+from social_core.exceptions import MissingBackend, AuthAlreadyAssociated
+from social_core.backends.oauth import BaseOAuth1, BaseOAuth2
+
+from authors.apps.core.permissions import IsAuthorOrReadOnly
+from authors.apps.authentication.backends import JWTAuthentication
 from . import models
```

Comment:
This has been imported in line 21. serializers have also been imported twice.

Reply:
Okay fixing it